packages/common/src/types.ts

```diff
@@ -24,6 +24,7 @@ export type MemoryUse = Opaque<number, 'MemoryUse'>;
 export type CPUUse = Opaque<number, 'CPUUse'>;
 export type BytesPerSecond = Opaque<number, 'BytesPerSecond'>;
 export type NetworkId = Opaque<string, 'NetworkId'>;
+export type NetworkState = string;
```

Comment:
Would keep it in line with `Opaque<string, 'NetworkState'>`, so that type inspection tells you what the value is (this is most useful for passing messages to the UI, which is done with plain arrays without field names).